product.json

```diff
@@ -3,6 +3,12 @@
   "appId": "io.podman_desktop.PodmanDesktop",
   "artifactName": "podman-desktop",
   "urlProtocol": "podman-desktop",
+  "defaultConfigFolder": "containers/podman-desktop",
+  "managedFolder": {
+    "macOS": "/Library/Application Support/Podman Desktop",
+    "windows": "%PROGRAMDATA%\\Podman Desktop",
+    "linux": "/usr/share/podman-desktop"
+  },
```

Comment:
shouldn't we do a nice nested objects ?

```json
"default": {
    "configFolder": ""
},
"managed-configuration": {
    "folders": "..."
}
```

I don't know what structure it should look likes, but IMO better avoid having everything at the root directory with complicated names

Reply:
I think I should remove default as anyway everything that is in this file are default values

then as it's for default paths value I would go with

```json
 "paths": {
      "config": "containers/podman-desktop",
      "managed": {
        "macOS": "/Library/Application Support/Podman Desktop",
        "windows": "%PROGRAMDATA%\\Podman Desktop",
        "linux": "/usr/share/podman-desktop"
      }
    }
```